getting-started/versioning.md

```diff
@@ -5,24 +5,24 @@ weight: 100
 
 # Versioning
 
+All components of this API have a single, common version. This document describe it's format and provide other information related to version.
```

Comment:
All components of this API have a single, common version. This document describe it's format and provide other information related to version. -> All components of this API have a single common version. This document describes its format and provides other related information.